README.md

```diff
@@ -57,63 +65,110 @@ This would result in a response with status code 404, and body
   "status_code": 404,
   "detail": {
     "message": "transaction not found",
-    "id": "some_id",
+    "id": "some_id"
   },
   "error_code": "A0001"
 }
 ```
-With this response, frontend can decide to simply render detail, or map it to detailed message.
-If error_code "A0001" correspond to the following i18 n entry
+
+With this response, frontend can decide to simply render detail, or map it to detailed message. If error_code "A0001"
+correspond to the following i18 n entry
+
 ```json
 "error.A0001": {"en-US": "transaction can not be found with supplied {id}: {message}"}
 ```
+
 we may format message accordingly with
+
 ```typescript
 errorMessage = formatMessage({ id: `error.${error.data.error_code}` }, error.data.detail);
 ```
 
-Note that error code is not supplied, is default to status code. So it is always safe to simply use error_code in frontend
-to decide what to render.
+Note that error code is not supplied, is default to status code. So it is always safe to simply use error_code in
+frontend to decide what to render.
 
 ### Data Store
+
 #### Minio
+
 refer to `tests/test_minio.py`
+
 #### Mongo
```

Comment:
can you help me to delete the section on odmantic, and change it to
refer to `tests/service/test_user.py`